<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>汽车销售管理系统</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>700</width>
     <height>410</height>
    </rect>
   </property>
   <widget class="QWidget" name="managerPage">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>561</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>品牌车管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>561</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>561</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>273</width>
           <height>273</height>
          </rect>
         </property>
         <attribute name="label">
          <string>出售车辆</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>12</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>厂商：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sellFactoryComboBox"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>品牌：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sellBrandComboBox"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>195</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>报价：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sellPriceLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>万</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>168</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>数量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sellNumSpinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="sellLastNumLabel">
             <property name="text">
              <string>剩余数量：000</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>195</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>金额：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sellSumLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>万</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>210</y>
            <width>158</width>
            <height>25</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="sellOkBtn">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="sellCancelBtn">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>273</width>
           <height>273</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="dailyList"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sellCancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>WidgetClass</receiver>
   <slot>on_sellCancelBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sellOkBtn</sender>
   <signal>clicked()</signal>
   <receiver>WidgetClass</receiver>
   <slot>on_sellOkBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_sellFactoryComboBox_currentIndexChanged(QString)</slot>
  <slot>on_sellCancelBtn_clicked()</slot>
  <slot>on_sellOkBtn_clicked()</slot>
 </slots>
</ui>
